core-web/libs/template-builder/src/lib/components/template-builder/models/models.ts

```diff
@@ -34,3 +35,7 @@ export enum TemplateBuilderBoxSize {
     medium = 'medium',
     small = 'small'
 }
+
+export interface StyleClassModel {
```

Comment:
Doc me

Reply:
Are you sure you want this here? looks very internal to me.